lineage/states/StateDistribution.py

```diff
@@ -94,16 +94,16 @@ def gamma_estimator(gamma_obs):
     """
     N = len(gamma_obs)
 
-    xbar = (sum(gamma_obs) + 7e-10) / (len(gamma_obs) + 1e-10)
+    xbar = (sum(gamma_obs) + 49e-10) / (len(gamma_obs) + 1e-10)
```

Comment:
Maybe `np.finfo(np.float64).eps` would be useful here?

Reply:
I wanted to change this in a different pull request, if that's OK with you.